safe-global/safe-wallet-web
# Fix: delay the notifications banner

## What it solves

It was popping up too fast overlapping with the onboarding modal.

<img width="525" alt="Screenshot 2023-11-20 at 11 20 38" src="https://github.com/safe-global/safe-wallet-web/assets/381895/1040092f-0763-44d4-90c5-88aa695af5f7">

## How to test
* Create a new Safe
* Wait till it's indexed
* Press the button to open the Safe
* Observe the onboarding help modal
* In 3 seconds, the notifications banner should appear
